ocaml/xapi/xapi_session.ml

```diff
@@ -415,8 +415,23 @@ let login_with_password ~__context ~uname ~pwd ~version ~originator = wipe_param
 					debug "Failed to locally authenticate user %s from %s: %s" uname (Context.get_origin __context) msg;
 					
 					(* 2. then against the external auth service *)
-					
-					(* 2.1. we first authenticate the user using the external authentication plugin *)
+					(* 2.1. we first check the external auth service status *)
+					let rec waiting_event_hook_auth_on_xapi_initialize_succeeded seconds =
+						if !Xapi_globs.event_hook_auth_on_xapi_initialize_succeeded then
+							0
```

Comment:
Thank you for the comments, I updated this PR and verified in trunk-ca-185733/108093.